docs/configuration_file.md

```diff
@@ -114,6 +114,21 @@ csv_file = <em>&lt;csv_file&gt;</em>
 Notes:
 * `account_nickname` is a user-selected identifier for the account;
 
+### Nexo (CSV)
+This plugin is CSV-based and parses CSV files generated by Nexo. Initialize it as follows:
+
+```
+[dali.plugin.input.csv.nexo <qualifiers>]
+account_holder = <account_holder>
+account_nickname = <account_nickname>
+transaction_csv_file = <csv_file>
+```
+
+Notes:
+
+* There is a transaction ID in the Nexo file, but it's an internal transaction ID so it ignored
```

Comment:
I think this is not relevant to the user: it's more targeted to developers. I would remove it from the user docs and maybe add it back as a comment in the code.

Reply:
Done!